### ISSUE TITLE
a11y: table structure and copy (first mini-pass)

### ISSUE THREAD

**Issue by candu** [CityofToronto/bdit_flashcrow#789]
**What Should Happen**
Tables should be described with a `<caption>` where possible, and any `<colgroup>` and `<rowgroup>` structure in them should be semantically valid.

**JIRA Tasks**
- [x] MOVE-170
- [x] table headings for requests
- [x] minor copy issues
